### PR TITLE
Add get(id) schedule operation

### PR DESCRIPTION
**Pull request by wobrak**
Add  Response mapper class for add schedule operation

### CODE REVIEW

File: TOZ_iOS/GetResponseMapper.swift [hunk 1]
```diff
@@ -0,0 +1,29 @@
+//
+//  GetResponseMapper.swift
+//  TOZ_iOS
+//
+//  Copyright © 2017 intive. All rights reserved.
+//
+
+import Foundation
+
+final class GetResponseMapper: ResponseMapper<ScheduleItem.ReservationItem>, ResponseMapperProtocol {
```
Comment (piotr-tobolski) [inline TOZ_iOS/GetResponseMapper.swift]: This name is too generic. Did you mean `GetScheduleResponseMapper`?

File: TOZ_iOS/GetResponseMapper.swift [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+            guard let startTime = json["startTime"] as? String else { return nil }
+            guard let endTime = json["endTime"] as? String else { return nil }
+            var timeOfDay: TimeOfDay
+            if startTime == "08:00" || endTime == "12:00" {
+                timeOfDay = TimeOfDay.morning
+            } else {
+                timeOfDay = TimeOfDay.afternoon
+            }
+            guard let ownerForename = json["ownerForename"] as? String else { return nil }
+            guard let ownerSurname = json["ownerSurname"] as? String else { return nil }
```
Comment (piotr-tobolski) [inline TOZ_iOS/GetResponseMapper.swift]: Both `ownerSurname` and `ownerForename` are declared as optional in Swagger model so they should be parsed `as? String?` and they should be optional in the model.

Comment (wobrak) [inline TOZ_iOS/GetResponseMapper.swift]: made optional, but they are required to make reservation or verify it.

File: TOZ_iOS/GetResponseMapper.swift (hunk 1 above)
Comment (wobrak) [inline TOZ_iOS/GetResponseMapper.swift]: name changed

**Pull request merged**